2021-11-14 23:22:56,669 DEBUG    root            Loaded Command Group: [u'gcloud', u'init']
2021-11-14 23:22:56,671 DEBUG    root            Running [gcloud.init] with arguments: []
2021-11-14 23:22:56,671 INFO     ___FILE_ONLY___ Welcome! This command will take you through the configuration of gcloud.


2021-11-14 23:22:56,674 INFO     ___FILE_ONLY___ Your current configuration has been set to: [default]


2021-11-14 23:22:56,674 INFO     ___FILE_ONLY___ You can skip diagnostics next time by using the following flag:

2021-11-14 23:22:56,674 INFO     ___FILE_ONLY___   gcloud init --skip-diagnostics


2021-11-14 23:22:56,674 INFO     ___FILE_ONLY___ Network diagnostic detects and fixes local network connection issues.

2021-11-14 23:22:56,675 INFO     ___FILE_ONLY___ Checking network connection...
2021-11-14 23:22:58,325 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): accounts.google.com:443
2021-11-14 23:22:58,455 DEBUG    urllib3.connectionpool https://accounts.google.com:443 "GET / HTTP/1.1" 302 338
2021-11-14 23:22:58,545 DEBUG    urllib3.connectionpool https://accounts.google.com:443 "GET /ServiceLogin?passive=1209600&continue=https%3A%2F%2Faccounts.google.com%2F&followup=https%3A%2F%2Faccounts.google.com%2F HTTP/1.1" 200 None
2021-11-14 23:22:58,617 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2021-11-14 23:22:58,744 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "GET /v1beta1/projects HTTP/1.1" 401 None
2021-11-14 23:22:58,749 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): www.googleapis.com:443
2021-11-14 23:22:58,865 DEBUG    urllib3.connectionpool https://www.googleapis.com:443 "GET /auth/cloud-platform HTTP/1.1" 200 14
2021-11-14 23:22:58,868 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): dl.google.com:443
2021-11-14 23:22:58,990 DEBUG    urllib3.connectionpool https://dl.google.com:443 "GET /dl/cloudsdk/channels/rapid/components-2.json HTTP/1.1" 200 169906
2021-11-14 23:22:59,190 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): expired.badssl.com:443
2021-11-14 23:22:59,550 DEBUG    urllib3.connectionpool https://expired.badssl.com:443 "GET / HTTP/1.1" 200 None
2021-11-14 23:22:59,712 INFO     ___FILE_ONLY___ Reachability Check passed.

2021-11-14 23:22:59,712 INFO     ___FILE_ONLY___ Network diagnostic passed (1/1 checks passed).


2021-11-14 23:23:05,342 DEBUG    root            Loaded Command Group: [u'gcloud', u'auth']
2021-11-14 23:23:05,522 DEBUG    root            Loaded Command Group: [u'gcloud', u'auth', u'login']
2021-11-14 23:23:05,524 INFO     root            Executing: [gcloud auth login --force --brief]
2021-11-14 23:23:05,526 DEBUG    root            Running [gcloud.auth.login] with arguments: [--brief: "True", --force: "True"]
2021-11-14 23:23:05,614 DEBUG    requests_oauthlib.oauth2_session Generated new state wgeTxu9m4dq9sNiP1LHPaWH1TnWM5Z.
2021-11-14 23:23:05,869 INFO     ___FILE_ONLY___ Your browser has been opened to visit:

    https://accounts.google.com/o/oauth2/auth?response_type=code&client_id=32555940559.apps.googleusercontent.com&redirect_uri=http%3A%2F%2Flocalhost%3A8085%2F&scope=openid+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fappengine.admin+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcompute+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Faccounts.reauth&state=wgeTxu9m4dq9sNiP1LHPaWH1TnWM5Z&code_challenge=3e6Favui9jzSCRyZhphuCz9kS1usDtY6-M9fFOTv3dE&code_challenge_method=S256&access_type=offline


2021-11-14 23:23:19,516 INFO     google_auth_oauthlib.flow "GET /?state=wgeTxu9m4dq9sNiP1LHPaWH1TnWM5Z&code=4/0AX4XfWgEm4WQ4ZlPwPK2b8YfIJyPc7N0PVkaxISj_7mUBw58Q92MRU60voPBJAWI5_yoNw&scope=email%20openid%20https://www.googleapis.com/auth/userinfo.email%20https://www.googleapis.com/auth/cloud-platform%20https://www.googleapis.com/auth/appengine.admin%20https://www.googleapis.com/auth/compute%20https://www.googleapis.com/auth/accounts.reauth&authuser=0&prompt=consent HTTP/1.1" 200 959
2021-11-14 23:23:19,517 DEBUG    requests_oauthlib.oauth2_session Requesting url https://oauth2.googleapis.com/token using method POST.
2021-11-14 23:23:19,517 DEBUG    requests_oauthlib.oauth2_session Supplying headers {'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8', 'Accept': 'application/json', 'user-agent': 'google-cloud-sdk gcloud/364.0.0 command/gcloud.auth.login invocation-id/74343ba27528448f8df4b8f339506168 environment/None environment-version/None interactive/True from-script/False python/2.7.16 term/xterm-256color (Macintosh; Intel Mac OS X 19.6.0)'} and data {u'code': u'4/0AX4XfWgEm4WQ4ZlPwPK2b8YfIJyPc7N0PVkaxISj_7mUBw58Q92MRU60voPBJAWI5_yoNw', u'code_verifier': u'8H-cWwmd0LksZ.DyNzglDBPRqaHxni2lvIOrZHImArwgZK1qketqUZyJvivhRfsq47bLiBSeQKFo4P82R7lv11X-aCw5cdnGLdgd3Xh5Rbd4eR99VCq3_56X04u7ARRh', u'redirect_uri': u'http://localhost:8085/', u'client_id': u'32555940559.apps.googleusercontent.com', u'client_secret': u'ZmssLNjJy2998hD4CTg2ejr2', u'grant_type': u'authorization_code'}
2021-11-14 23:23:19,518 DEBUG    requests_oauthlib.oauth2_session Passing through key word arguments {'verify': True, 'json': None, 'proxies': None, 'timeout': None, 'auth': None}.
2021-11-14 23:23:19,564 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): oauth2.googleapis.com:443
2021-11-14 23:23:19,817 DEBUG    urllib3.connectionpool https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
2021-11-14 23:23:19,818 DEBUG    requests_oauthlib.oauth2_session Prepared fetch token request body grant_type=authorization_code&client_id=32555940559.apps.googleusercontent.com&client_secret=ZmssLNjJy2998hD4CTg2ejr2&code_verifier=8H-cWwmd0LksZ.DyNzglDBPRqaHxni2lvIOrZHImArwgZK1qketqUZyJvivhRfsq47bLiBSeQKFo4P82R7lv11X-aCw5cdnGLdgd3Xh5Rbd4eR99VCq3_56X04u7ARRh&code=4%2F0AX4XfWgEm4WQ4ZlPwPK2b8YfIJyPc7N0PVkaxISj_7mUBw58Q92MRU60voPBJAWI5_yoNw&redirect_uri=http%3A%2F%2Flocalhost%3A8085%2F
2021-11-14 23:23:19,818 DEBUG    requests_oauthlib.oauth2_session Request to fetch token completed with status 200.
2021-11-14 23:23:19,819 DEBUG    requests_oauthlib.oauth2_session Request headers were {'Content-Length': '387', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'user-agent': 'google-cloud-sdk gcloud/364.0.0 command/gcloud.auth.login invocation-id/74343ba27528448f8df4b8f339506168 environment/None environment-version/None interactive/True from-script/False python/2.7.16 term/xterm-256color (Macintosh; Intel Mac OS X 19.6.0)', 'Connection': 'keep-alive', 'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'}
2021-11-14 23:23:19,819 DEBUG    requests_oauthlib.oauth2_session Request body was code=4%2F0AX4XfWgEm4WQ4ZlPwPK2b8YfIJyPc7N0PVkaxISj_7mUBw58Q92MRU60voPBJAWI5_yoNw&code_verifier=8H-cWwmd0LksZ.DyNzglDBPRqaHxni2lvIOrZHImArwgZK1qketqUZyJvivhRfsq47bLiBSeQKFo4P82R7lv11X-aCw5cdnGLdgd3Xh5Rbd4eR99VCq3_56X04u7ARRh&redirect_uri=http%3A%2F%2Flocalhost%3A8085%2F&client_id=32555940559.apps.googleusercontent.com&client_secret=ZmssLNjJy2998hD4CTg2ejr2&grant_type=authorization_code
2021-11-14 23:23:19,819 DEBUG    requests_oauthlib.oauth2_session Response headers were {'X-XSS-Protection': '0', 'X-Content-Type-Options': 'nosniff', 'Content-Encoding': 'gzip', 'Transfer-Encoding': 'chunked', 'Expires': 'Mon, 01 Jan 1990 00:00:00 GMT', 'Vary': 'Origin, X-Origin, Referer', 'Server': 'scaffolding on HTTPServer2', 'Pragma': 'no-cache', 'Cache-Control': 'no-cache, no-store, max-age=0, must-revalidate', 'Date': 'Mon, 15 Nov 2021 07:23:19 GMT', 'X-Frame-Options': 'SAMEORIGIN', 'Alt-Svc': 'clear', 'Content-Type': 'application/json; charset=utf-8'} and content {
  "access_token": "ya29.a0ARrdaM-T-Tj_0FukHpXFwh0-eAOMWg-cp2tS-X9nXW3E0m-N6kNT5vp77hSKKvWWZe2e-KhQt1JAAqlXe5tWbgAP79TycjahxuVd7LVelbAarB4-gVqZ6IBe8kvbJWPYsoNDvQXcFKS9pfrqE92oMDRqsXGIPQY6upra",
  "expires_in": 3599,
  "refresh_token": "1//06tIgAGwH0AOzCgYIARAAGAYSNwF-L9IrZRJGJHafFbTPyzJO6pVDkocB43f0piFdD4rFcpAO3-HRd2NFCE2n_aY154GNV5UmzpA",
  "scope": "openid https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/appengine.admin https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/compute https://www.googleapis.com/auth/accounts.reauth",
  "token_type": "Bearer",
  "id_token": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjI3YzcyNjE5ZDA5MzVhMjkwYzQxYzNmMDEwMTY3MTM4Njg1ZjdlNTMiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL2FjY291bnRzLmdvb2dsZS5jb20iLCJhenAiOiIzMjU1NTk0MDU1OS5hcHBzLmdvb2dsZXVzZXJjb250ZW50LmNvbSIsImF1ZCI6IjMyNTU1OTQwNTU5LmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTAxNDIwNTIzNTk3NzQ0Mjk1MDYyIiwiZW1haWwiOiJ6YWNoc3RhbWVnbmFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiI0ektqSWUwbXdfVkVHUjk1WkpCTUdnIiwiaWF0IjoxNjM2OTYwOTk5LCJleHAiOjE2MzY5NjQ1OTl9.e0bOlCT5Qzvp20Ei8se9qHE7cT7B0Pjyn-esrFDDwhirCs8SjRAnek9zY5WHdzB3m4vLFHKddgq9EkeMKrf44T6f4gxJrs4PladCmMyHa4YqFQ883909SndRQuCJ264nOMsnzHDR5tMcl1frBYuyWmg0xoVbpfyCOrT7yT4NYlIn0m9GbrwaaviagzOXaOHdrhfeD58dPK3j2ldMtRLHT6_rMvptqTQMhVTrdIkth5WRrfljIwsiDtIonKMUNWSUrYE5CMq1cqM9scdCZJ2soOc6Y7lrc-StHZcM51jhCPrXMmpLOf735CLqAfJu_GWMLDEqNTgNQVNYsyGP6BrVMg"
}.
2021-11-14 23:23:19,819 DEBUG    requests_oauthlib.oauth2_session Invoking 0 token response hooks.
2021-11-14 23:23:19,820 DEBUG    requests_oauthlib.oauth2_session Obtained token {u'token_type': u'Bearer', u'refresh_token': u'1//06tIgAGwH0AOzCgYIARAAGAYSNwF-L9IrZRJGJHafFbTPyzJO6pVDkocB43f0piFdD4rFcpAO3-HRd2NFCE2n_aY154GNV5UmzpA', u'scope': [u'openid', u'https://www.googleapis.com/auth/cloud-platform', u'https://www.googleapis.com/auth/appengine.admin', u'https://www.googleapis.com/auth/userinfo.email', u'https://www.googleapis.com/auth/compute', u'https://www.googleapis.com/auth/accounts.reauth'], u'access_token': u'ya29.a0ARrdaM-T-Tj_0FukHpXFwh0-eAOMWg-cp2tS-X9nXW3E0m-N6kNT5vp77hSKKvWWZe2e-KhQt1JAAqlXe5tWbgAP79TycjahxuVd7LVelbAarB4-gVqZ6IBe8kvbJWPYsoNDvQXcFKS9pfrqE92oMDRqsXGIPQY6upra', u'id_token': u'eyJhbGciOiJSUzI1NiIsImtpZCI6IjI3YzcyNjE5ZDA5MzVhMjkwYzQxYzNmMDEwMTY3MTM4Njg1ZjdlNTMiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL2FjY291bnRzLmdvb2dsZS5jb20iLCJhenAiOiIzMjU1NTk0MDU1OS5hcHBzLmdvb2dsZXVzZXJjb250ZW50LmNvbSIsImF1ZCI6IjMyNTU1OTQwNTU5LmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTAxNDIwNTIzNTk3NzQ0Mjk1MDYyIiwiZW1haWwiOiJ6YWNoc3RhbWVnbmFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiI0ektqSWUwbXdfVkVHUjk1WkpCTUdnIiwiaWF0IjoxNjM2OTYwOTk5LCJleHAiOjE2MzY5NjQ1OTl9.e0bOlCT5Qzvp20Ei8se9qHE7cT7B0Pjyn-esrFDDwhirCs8SjRAnek9zY5WHdzB3m4vLFHKddgq9EkeMKrf44T6f4gxJrs4PladCmMyHa4YqFQ883909SndRQuCJ264nOMsnzHDR5tMcl1frBYuyWmg0xoVbpfyCOrT7yT4NYlIn0m9GbrwaaviagzOXaOHdrhfeD58dPK3j2ldMtRLHT6_rMvptqTQMhVTrdIkth5WRrfljIwsiDtIonKMUNWSUrYE5CMq1cqM9scdCZJ2soOc6Y7lrc-StHZcM51jhCPrXMmpLOf735CLqAfJu_GWMLDEqNTgNQVNYsyGP6BrVMg', u'expires_in': 3599, u'expires_at': 1636964598.820062}.
2021-11-14 23:23:19,827 INFO     root            Display format: "none"
2021-11-14 23:23:20,034 DEBUG    root            Checking for updates...
2021-11-14 23:23:20,037 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): dl.google.com:443
2021-11-14 23:23:20,151 DEBUG    urllib3.connectionpool https://dl.google.com:443 "GET /dl/cloudsdk/channels/rapid/components-2.json HTTP/1.1" 200 169906
2021-11-14 23:23:20,287 DEBUG    root            Updating notification cache...
2021-11-14 23:23:20,290 INFO     ___FILE_ONLY___ You are logged in as: [zachstamegna@gmail.com].


2021-11-14 23:23:20,332 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2021-11-14 23:23:20,555 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "GET /v1/projects?filter=lifecycleState%3AACTIVE&alt=json&pageSize=201 HTTP/1.1" 200 None
2021-11-14 23:25:36,936 INFO     ___FILE_ONLY___ Your current project has been set to: [high-comfort-332206].


2021-11-14 23:25:37,081 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute']
2021-11-14 23:25:37,085 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute', u'project_info']
2021-11-14 23:25:37,123 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute', u'project-info', u'describe']
2021-11-14 23:25:37,124 INFO     root            Executing: [gcloud compute project-info describe --quiet]
2021-11-14 23:25:37,126 DEBUG    root            Running [gcloud.compute.project-info.describe] with arguments: [--no-user-output-enabled: "false", --quiet: "True", --verbosity: "none"]
2021-11-14 23:25:37,605 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2021-11-14 23:25:37,891 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /batch/compute/v1 HTTP/1.1" 200 None
2021-11-14 23:25:37,921 INFO     root            Display disabled.
2021-11-14 23:25:52,576 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute', u'zones']
2021-11-14 23:25:52,578 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute', u'zones', u'list']
2021-11-14 23:25:52,580 INFO     root            Executing: [gcloud compute zones list]
2021-11-14 23:25:52,586 DEBUG    root            Running [gcloud.compute.zones.list] with arguments: [--no-user-output-enabled: "false", --verbosity: "none"]
2021-11-14 23:25:52,605 INFO     root            Display disabled.
2021-11-14 23:25:52,608 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2021-11-14 23:25:53,070 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /batch/compute/v1 HTTP/1.1" 200 None
2021-11-14 23:27:13,971 INFO     ___FILE_ONLY___ Your project default Compute Engine zone has been set to [us-central1-f].
You can change it by running [gcloud config set compute/zone NAME].


2021-11-14 23:27:13,978 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute', u'regions']
2021-11-14 23:27:13,991 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute', u'regions', u'describe']
2021-11-14 23:27:13,993 INFO     root            Executing: [gcloud compute regions describe https://www.googleapis.com/compute/v1/projects/high-comfort-332206/regions/us-central1]
2021-11-14 23:27:13,998 DEBUG    root            Running [gcloud.compute.regions.describe] with arguments: [--no-user-output-enabled: "false", --verbosity: "none", NAME: "https://www.googleapis.com/compute/v1/projects/high-comfort-332206/regions/us-central1"]
2021-11-14 23:27:14,025 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2021-11-14 23:27:14,472 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /batch/compute/v1 HTTP/1.1" 200 None
2021-11-14 23:27:14,487 INFO     root            Display disabled.
2021-11-14 23:27:14,500 INFO     ___FILE_ONLY___ Your project default Compute Engine region has been set to [us-central1].
You can change it by running [gcloud config set compute/region NAME].


2021-11-14 23:27:14,501 DEBUG    root            Executing command: [u'/Users/zachstamegna/google-cloud-sdk/bin/gsutil', u'config', u'-n', u'-o', u'/Users/zachstamegna/.boto']
2021-11-14 23:27:14,522 DEBUG    root            (gcloud.init) /Users/zachstamegna/google-cloud-sdk/bin/gsutil: command not found
Traceback (most recent call last):
  File "/Users/zachstamegna/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/Users/zachstamegna/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 809, in Run
    resources = command_instance.Run(args)
  File "/Users/zachstamegna/google-cloud-sdk/lib/surface/init.py", line 152, in Run
    self._CreateBotoConfig()
  File "/Users/zachstamegna/google-cloud-sdk/lib/surface/init.py", line 454, in _CreateBotoConfig
    err_func=log.file_only_logger.debug)
  File "/Users/zachstamegna/google-cloud-sdk/lib/googlecloudsdk/core/execution_utils.py", line 384, in Exec
    **extra_popen_kwargs)
  File "/Users/zachstamegna/google-cloud-sdk/lib/googlecloudsdk/core/execution_utils.py", line 311, in _Exec
    raise InvalidCommandError(args[0])
InvalidCommandError: /Users/zachstamegna/google-cloud-sdk/bin/gsutil: command not found
2021-11-14 23:27:14,527 ERROR    root            (gcloud.init) /Users/zachstamegna/google-cloud-sdk/bin/gsutil: command not found
